.ci/lib.sh

```diff
@@ -35,7 +35,9 @@ function clone_and_build() {
 
 	# Override branch if we are testing a PR.
```

Comment:
please update this comment

Reply:
not sure what you mean, but comment is still valid

Reply:
if it's valid, it's valid :smile: